### PR TITLE
Add task 7

### PR DESCRIPTION
**Pull request by Garrett-Donowan**

### CODE REVIEW

File: Tasks/Gorbachev/Task_7/bl.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import data
+from decimal import Decimal
+
+def format(catalog_dict, sale=False, category=None): #todo better solution
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Gorbachev/Task_7/bl.py:4]: an interesting comment

File: Tasks/Gorbachev/Task_7/data.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        return category_dict[choosen_category]
+    else:
+        return False
+
+def buy_items():
+    """Return empty basket and decreased quantity of items in catalog"""
+    global catalog_dict, basket_dict
+    if basket_dict == {}:
+        return False
+    for y,z in catalog_dict.copy().items():
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Gorbachev/Task_7/data.py:46]: are you sure the simple copy is enough?

**Review by MikitaTsiarentsyeu — APPROVED** [review]
this is very good, all the logic are correctly distributed across the layers, the score is 10

File: Tasks/Gorbachev/Task_7/data.py (hunk 2 above)
Comment (Garrett-Donowan) [reply Tasks/Gorbachev/Task_7/data.py:46]: we search in copy of initial catalog. and when quantity of goods is 0 we need to delete all position from initial catalog. if we'll serch in initial catalog in loop we cannot delete position in initial catalog.
maybe there is better solution.

**Pull request merged**